### PR TITLE
Add missing release notes for chef-client 13.12 / 14.6

### PR DESCRIPTION
**Pull request by tas50**
Fixup some formatting while I'm in here as well

### CODE REVIEW

File: chef_master/source/release_notes.rst [hunk 1]
```diff
@@ -1,10 +1,106 @@
...
+
+* **Smaller Package and Install Size**
+
+  Both Chef packages and on disk installations have been greatly reduced in size by trimming unnecessary installation files. This has reduced our package size on macOS/Linux by ~50% and Windows by ~12%. With this change Chef 14 is now smaller than a legacy Chef 10 package.
+
+* **New Resources**
+
+  * **timezone**
+
+    Chef now includes the `timezone </resource_timezone.html>`__ resource from `@dragonsmith <http://github.com/dragonsmith>`__'s ``timezone_lwrp`` cookbook. This resource supports setting a Linux node's timezone. Thank you `@dragonsmith <http://github.com/dragonsmith>`__'s for allowing us to include this out of the box in Chef.
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst]: Thank you `@dragonsmith <http://github.com/dragonsmith>`__ for allowing us to include this in Chef.

File: chef_master/source/release_notes.rst [hunk 2]
```diff
@@ -1,10 +1,106 @@
...
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/release_notes.rst>`__
 
 Chef Client is released on a monthly schedule with new releases the first Wednesday of every month. Below are the major changes for each release. For a detailed list of changes see the `Chef changelog <https://github.com/chef/chef/blob/master/CHANGELOG.md>`__
 
+What’s New in 14.6
+=====================================================
+
+* **Smaller Package and Install Size**
+
+  Both Chef packages and on disk installations have been greatly reduced in size by trimming unnecessary installation files. This has reduced our package size on macOS/Linux by ~50% and Windows by ~12%. With this change Chef 14 is now smaller than a legacy Chef 10 package.
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst]: We trimmed unnecessary installation files, greatly reducing the sizes of both Chef packages and on disk installations. macOS/Linux packages are ~50% smaller and Windows packages are ~12% smaller. Chef 14 is now smaller than a legacy Chef 10 package.

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: chef_master/source/release_notes.rst (hunk 2 above)
Comment (tas50) [reply chef_master/source/release_notes.rst]: Updated

File: chef_master/source/release_notes.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/release_notes.rst]: Updated

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**